File: Samples/AppKit/Objective-C/WindowController.m
//
//  WindowController.m
//  SetappSamples
//
//  Created by Сергій Попов on 14.04.2023.
//

#import "WindowController.h"

@interface WindowController ()

@end

@implementation WindowController

- (IBAction)showHelp:(id)sender {
    [NSWorkspace.sharedWorkspace openURL:[NSURL URLWithString:@"https://docs.setapp.com"]];
}

@end

File: Samples/AppKit/Swift/WindowController.swift
//
//  WindowController.swift
//  SetappSamples
//
//  Created by Сергій Попов on 14.04.2023.
//

import AppKit

class WindowController: NSWindowController {

    @IBAction func showHelp(_ sender: Any) {
        if let setappURL = URL(string: "https://docs.setapp.com") {
            NSWorkspace.shared.open(setappURL)
        }
    }
    
}
